To workaround the following error

  $ clang conftest.c -I/usr/local/include/xorg
  In file included from conftest.c:3:
  In file included from /usr/local/include/xorg/xf86Optionstr.h:27:
  /usr/local/include/xorg/list.h:216:15: error: unknown type name 'Bool'; did you mean
        '_Bool'?
  static inline Bool
                ^~~~
                _Bool
  1 error generated.

  $ gcc conftest.c -I/usr/local/include/xorg
  In file included from /usr/local/include/xorg/xf86Optionstr.h:27,
                   from conftest.c:3:
  /usr/local/include/xorg/list.h:217: error: expected '=', ',', ';', 'asm' or '__attribute__' before 'xorg_list_is_empty'

cf. http://cgit.freedesktop.org/xorg/xserver/commit/?id=13b5074

--- configure~
+++ configure
@@ -18788,6 +18788,7 @@ CFLAGS="$XORG_CFLAGS $CFLAGS"
 cat confdefs.h - <<_ACEOF >conftest.$ac_ext
 /* end confdefs.h.  */
 
+	  typedef int Bool;
 	  #include <xf86Optionstr.h>
 int
 main ()
